src/utils/util.js

```diff
@@ -11,139 +12,140 @@ const util = require('util');
  * @public
  * @type {Utils}
  */
-export default class Utils {
-  /**
-   * Get the lowest SmartRate from a provided list of SmartRates.
-   * @public
-   * @param {Rate[]} smartrates - List of SmartRates to filter through
-   * @param {number} deliveryDays - The maximum number of days allowed for delivery
-   * @param {string} deliveryAccuracy - The target level of accuracy for the delivery days (e.g. 'percentile_50')
-   * @returns {Rate} - The lowest SmartRate
-   * @throws {FilteringError} - If no applicable rates are found
-   * @throws {InvalidParameterError} - If the deliveryAccuracy value is invalid
-   */
-  static getLowestSmartRate(smartrates, deliveryDays, deliveryAccuracy) {
-    const validDeliveryAccuracyValues = new Set([
-      'percentile_50',
-      'percentile_75',
-      'percentile_85',
-      'percentile_90',
-      'percentile_95',
-      'percentile_97',
-      'percentile_99',
-    ]);
-    let lowestSmartRate = null;
-    const lowercaseDeliveryAccuracy = deliveryAccuracy.toLowerCase();
-
-    if (!validDeliveryAccuracyValues.has(lowercaseDeliveryAccuracy)) {
-      throw new InvalidParameterError({
-        message: `Invalid deliveryAccuracy value, must be one of: ${new Array(
-          ...validDeliveryAccuracyValues,
-        ).join(', ')}`,
-      });
-    }
+export default (easypostClient) =>
+  class Utils extends baseService(easypostClient) {
```

Comment:
issue: this should not extend the `base_service` because we don't need anything from the base service here since none of these are making API calls.